mautic/mautic
# do no update back to Mautic with 'Unknown'

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
When updating blank values do not update back to Mautic with Unknown.
[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Sync a contact to Salesforce with company blank. you should have update blank values selected in your configuration for Salesforce
2. This should be written into SF with the word Unknown.
3. Sync one more time, the contact will have it's company written back with Unknown. which should not happen.

#### Steps to test this PR:
1. Apply this PR, remove the company unknown from the mautic contact
2. Sync again and the Mautic contact should continue with it's field as blank. it should be the same for last name and all required fields.

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
